### PR TITLE
geonkick: 2.6.1 -> 2.8.0

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -17107,8 +17107,6 @@ in
 
   randomx = callPackage ../development/libraries/randomx { };
 
-  redkite = callPackage ../development/libraries/redkite { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:17110]: Please add an alias to aliases.nix.

Comment (magnetophon) [reply pkgs/top-level/all-packages.nix:17110]: You mean a message saying it's been removed?
Redkite was written for geonkick so I'm pretty sure no one has or wants it installed standalone.

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:17110]: > You mean a message saying it's been removed?

yes.

> Redkite was written for geonkick so I'm pretty sure no one has or wants it installed standalone.

Please do it anyway. :)

Comment (magnetophon) [reply pkgs/top-level/all-packages.nix:17110]: Here you go! :)

**Pull request merged**